README.md

```diff
@@ -537,7 +537,7 @@ get more information and examples on how to use it.
 
 This is the author's first try at writing Rust code. He learned a lot in the
 process, but there remains a feeling that this crate has a kind of C-ish style
-in some places instead of the Rusty look the author would like it to have. So
+in some places instead of the Rusty look, the author would like it to have. So
```

Comment:
Nack for this one, it's “the author would like it to have the Rusty look”, adding a comma does not sound right.